packages/mobile/src/verify/__snapshots__/VerificationLoadingScreen.test.tsx.snap

```diff
@@ -278,15 +278,15 @@ exports[`VerificationLoadingScreen renders correctly 1`] = `
         <Text
```

Comment:
Minor changes to modal because modal component is used- note that this will change when redesign is applied